<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>985</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyWidget" name="ekran">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label21">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation:</string>
    </property>
   </widget>
   <widget class="QSlider" name="translationYSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>130</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotationXSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>350</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="shearingXSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>480</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label14">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>130</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label12">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QSlider" name="shearingYSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>510</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>510</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label22">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>190</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaling:</string>
    </property>
   </widget>
   <widget class="QSlider" name="scalingYSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>250</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label13_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>320</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label15">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QSlider" name="scalingXSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>220</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>480</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label13">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label152">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>440</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Shearing:</string>
    </property>
   </widget>
   <widget class="QSlider" name="translationXSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>100</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>160</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QSlider" name="translationZSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>160</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>280</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QSlider" name="scalingZSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>280</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>380</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QSlider" name="rotationYSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>380</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>410</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QSlider" name="rotationZSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>410</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label7_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>540</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QSlider" name="shearingZSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>540</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1413</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
